pkg/server/server.go

```diff
@@ -459,7 +468,9 @@ func (s *ProxyServer) serveRecvFrontend(stream client.ProxyService_ProxyServer,
 			if err := backend.Send(pkt); err != nil {
 				// TODO: retry with other backends connecting to this agent.
 				klog.ErrorS(err, "CLOSE_REQ to Backend failed", "serverID", s.serverID, "connectionID", connID)
+				continue
```

Comment:
It might be slightly easier to understand this by using an else block here to set closeReqSent

Reply:
Sure, I could do that, I was just following the generic error handling pattern in the function :) Thank you!